README.md

```diff
@@ -110,32 +126,31 @@ Another purpose is to serve as an example of how to build an Event Source using
      --role roles/pubsub.publisher
    ```
 
-   1. Create a namespace for where the secret is created and where our controller will run
-
-      ```shell
-      kubectl create namespace gcssource-system
-      ```
+   1. Download a new JSON private key for that Service Account. **Be sure not to
+      check this key into source control!**
 
-   1. Create a secret on the kubernetes cluster for the downloaded key. You need
-      to store this key in `key.json` in a secret named `gcssource-key`. This is used by the
-	  `controller` to create GCS notifications and pubsub topics.
+   ```shell
+   gcloud iam service-accounts keys create gcs-source.json \
+     --iam-account=knative-source@$PROJECT_ID.iam.gserviceaccount.com
+   ```
 
-      ```shell
-      kubectl -n gcssource-system create secret generic gcssource-key --from-file=key.json=gcs-source.json
-      ```
+1. Create two secrets on the kubernetes cluster with the downloaded key:
 
-      The name `gcssource-key` and `key.json` are pre-configured values
-      in the controller which manages your Cloud Storage sources.
+   ```shell
+   # Note that the first secret may already have been created when installing
+   # Knative Eventing. The following command will overwrite it. If you don't
+   # want to overwrite it, then skip this command.
+   kubectl --namespace gcssource-system create secret generic gcs-source-key --from-file=key.json=gcs-source.json --dry-run --output yaml | kubectl apply --filename -
```

Comment:
why change this like so? in case it already exists?

Reply:
This is the gcs-source key so I don't see this actually existing. I can see the one below existing if pubsub has already been created?

Reply:
I copied it from the updated docs in knative/docs: https://github.com/knative/docs/blob/8d62a0b0663a8f2b91e174f805b1837b6ec108ec/docs/eventing/samples/gcp-pubsub-source/README.md

Reply:
I see what you are saying, yeah that needs to be cleaned.. one sec

Reply:
reverted it to just create the key